src/lib/pages/backoffice/Loan/LoanDetails/LoanOverbookedWarning/LoanOverbookedWarning.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withCancel } from '@api/utils';
+import { documentApi } from '@api/documents';
+
+export default class LoanOverbookedWarning extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      overbooked: false,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchCirculation()
+  }
+
+  componentWillUnmount() {
+    this.cancellableCirculation && this.cancellableCirculation.cancel();
+  }
+
+  fetchCirculation = async () => {
+    const { documentPid } = this.props;
+    try {
+      this.cancellableCirculation = withCancel(
+        documentApi.get(documentPid)
+      );
+      const response = await this.cancellableCirculation.promise;
+      const circulation = response.data.metadata.circulation;
+      this.setState({ overbooked: circulation.overbooked});
+    } catch (error) {
+      if (error !== 'UNMOUNTED') {
+        this.setState({ overbooked: false});
```

Comment:
During yesterday stand up, @kprzerwa was suggesting to display an error message on error instead.

Example:
```javascript
OVERBOOKED_MSG = "This literature is overbooked."
OVERBOOKED_ERROR = "Cannot verify if this literature is overbooked. Please refresh the page."

export default class LoanOverbookedWarning extends Component {
...
      this.setState({ msg: circulation.overbooked ? OVERBOOKED_MSG : ''});
    } catch (error) {
      if (error !== 'UNMOUNTED') {
          this.setState({ msg: OVERBOOKED_ERROR});
```